server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfigurationTest.java

```diff
@@ -33,6 +33,7 @@
 
 public class CassandraCacheConfigurationTest {
 
+    public static final int IN_FLIGHT_REQUESTS_THRESHOLD = 562;
```

Comment:
Why this special number?

Reply:
Because it is different than the default. Why?